packages/realm/scripts/build-ios.sh

```diff
@@ -97,15 +98,15 @@ DEVELOPER_DIR="${DEVELOPER_DIR:-${SELECTED_DEVELOPER_DIR}}"
 
 # Configure CMake project
 SDKROOT="$DEVELOPER_DIR/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/" cmake "$BINDGEN_PATH" -GXcode \
-    -DCMAKE_TOOLCHAIN_FILE="$PROJECT_ROOT/packages/realm/bindgen/vendor/realm-core/tools/cmake/xcode.toolchain.cmake" \
+    -DCMAKE_TOOLCHAIN_FILE="$BINDGEN_PATH/vendor/realm-core/tools/cmake/xcode.toolchain.cmake" \
     -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY="$(pwd)/out/$<CONFIG>\$EFFECTIVE_PLATFORM_NAME" \
 
 DEVELOPER_DIR="$DEVELOPER_DIR" xcodebuild build \
     -scheme realm-js-ios \
     "${DESTINATIONS[@]}" \
     -configuration $CONFIGURATION \
-    CC="$PROJECT_ROOT/scripts/ccache-clang.sh" \
-    CXX="$PROJECT_ROOT/scripts/ccache-clang++.sh" \
+    CC="$PROJECT_ROOT/../../scripts/ccache-clang.sh" \
+    CXX="$PROJECT_ROOT/../../scripts/ccache-clang++.sh" \
```

Comment:
```suggestion
    CC="$PROJECT_ROOT/scripts/ccache-clang.sh" \
    CXX="$PROJECT_ROOT/scripts/ccache-clang++.sh" \
```

I don't think these needs an update?

Reply:
It does need this.  I'll just merge it in its working form right now.  The build optimizations for iOS will overwrite this anyway.